api_app/db/repositories/operations.py

```diff
@@ -96,7 +96,7 @@ def build_step_list(self, steps: List[OperationStep], resource_template_dict: di
 
                         # if it's a shared service, should be a singleton across the TRE, get it by template name
                         if step["resourceType"] == ResourceType.SharedService:
-                            resource_for_step = resource_repo.get_resource_by_template_name(step["resourceTemplateName"])
+                            resource_for_step = resource_repo.get_operating_resource_by_template_name(step["resourceTemplateName"])
```

Comment:
yes, I followed the same naming like in [SharedServiceRepository](https://github.com/microsoft/AzureTRE/blob/main/api_app/db/repositories/shared_services.py#L32)
I can change both to active if it make more sense

Reply:
Is the original developer still with us and we can check with them?
I support renaming it but it's not a must.

Reply:
I think it was me! I think I was trying to avoid it sounding too similar to `isEnabled`, but if you think that `active` makes no sense, by all means, please rename :)

Reply:
I've renamed all appearances of `operating` to `active`.